server/.env.example

```diff
@@ -15,3 +15,8 @@ GOOGLE_CLIENT_SECRET = 'Xtktwt99ZXrL8rIVt23aStU2'
 GOOGLE_CALLBACK_URL = 'http://localhost:8080/auth/google/callback'
 
 EXPIRES='3600'
+
+AWS_SECRET_ACCESS_KEY = 
+AWS_ACCESS_KEY_ID = 
+AWS_REGION = 
```

Comment:
?

Reply:
Not sure how to change it. Changed to AWS_SECRET_ACCESS_KEY =AWS_SECRET_ACCESS_KEY